health-services/ExerciseSampleCompose/app/src/test/java/com/example/exercisesamplecompose/presentation/PreparingExerciseScreenTest.kt

```diff
@@ -52,15 +44,10 @@ class PreparingExerciseScreenTest(override val device: WearDevice) :
                 )
             )
         }
-    }
 
     @Test
     fun failed() = runTest {
-        AppScaffold(
```

Comment:
IF you make the change to clear on timeText via ScreenScaffold, you can leave this.

Reply:
Adjusted back